test/webworker/webworker-script.js

```diff
@@ -4,7 +4,7 @@
 
 // Abort if we are not running in a WebWorker
 if (typeof importScripts !== "undefined") {
-    importScripts("../../pkg/sinon.js");
+    importScripts("https://localhost:8080/pkg/sinon.js");
```

Comment:
shouldn't this be just `"/pkg/sinon.js"`?

Reply:
Good point. Adjusted in b3cd2d81.